File: baekjoon_all/10000+/boj_10989.py
# Solve 2022-05-30
# Update 2025-09-05

import sys

input = lambda: sys.stdin.readline().rstrip()

cnt = [0] * 10001

for _ in range(int(input())):
    cnt[int(input())] += 1

for x in range(10001):
    for _ in range(cnt[x]):
        print(x)

File: baekjoon_all/10000+/boj_10989.java
// Solve 2024-01-29

// 백준에 제출할 때는 class 이름을 Main으로 설정해야 한다.

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class boj_10989 {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        int n = Integer.parseInt(br.readLine());
        int[] cnt = new int[10001];

        for (int i = 0; i < n; i++) {
            int num = Integer.parseInt(br.readLine());
            cnt[num]++;
        }

        for (int num = 1; num <= 10000; num++) {
            int iter = cnt[num];

            while (iter-- > 0) {
                sb.append(num).append("\n");
            }
        }

        System.out.print(sb);
        br.close();
    }

}
